readme.md

```diff
@@ -29,16 +29,19 @@ console.log(results);
 /*
 [
 	{
+		status: 'fulfilled',
 		isFulfilled: true,
 		isRejected: false,
 		value: '🦄'
```

Comment:
Can you move `value` below `status` everywhere?

Reply:
PTAL